bcgov/moh-prime
# PRIME-2125 remove Site ID from obo site page for enrollees

[PRIME-2125](https://bcgovmoh.atlassian.net/browse/PRIME-2125)

### How to test:

1. Create a normal enrolment, 
2. Don't provide a certificate
3. When you go to Obo Sites page you will not see the Site ID field under the address (same for paper enrollee)
4. Continue submission to make sure nothing broke along the way
